scripts/globals/mobskills/Numbing_Noise.lua

```diff
@@ -12,6 +12,19 @@ require("/scripts/globals/status");
 require("/scripts/globals/monstertpmoves");
 
 ---------------------------------------------
+-- For Race Runner
+local path = {
```

Comment:
i don't like the idea of shoving NM specific stuff into a mobskill script, all mob specific stuff should be in the mobs script

if you can't do that, then I will make a binding for it